bin/install.js

```diff
@@ -1,22 +1,22 @@
 /* eslint-disable no-console */
 "use strict";
 
-var exec = require("child_process").exec;
-var path = require("path");
+const exec = require("child_process").exec;
+const path = require("path");
 
 
-var MAX_ATTEMPTS = process.env.SERVER_INSTALL_ATTEMPTS || 5;
-var INTERVAL = 1500;
-var attemptedToBuild = false;
+const MAX_ATTEMPTS = process.env.SERVER_INSTALL_ATTEMPTS || 5;
+const INTERVAL = 1500;
+let attemptedToBuild = false;
 
 function doInstall () {
   // UiAutomator2 needs Java. Fail early if it doesn't exist
-  var androidHelpers = require('appium-android-driver').androidHelpers;
-  androidHelpers.getJavaVersion().then(function () {
-    var tries = 0;
+  let androidHelpers = require('appium-android-driver').androidHelpers;
+  androidHelpers.getJavaVersion().then(function () { // eslint-disable-line promise/prefer-await-to-then
```

Comment:
should we make this exception file-wide since we can't do promises in this whole file?